src/components/user-steps-wrapper/UserStepsWrapper.jsx

```diff
@@ -44,13 +46,11 @@ const UserStepsWrapper = ({ userRole }) => {
       key='1'
       setIsUserFetched={setIsUserFetched}
     />,
-    <SubjectsStep key='2' />,
+    <SubjectsStep key='2' stepLabels={stepLabels} />,
```

Comment:
we already have `stepLabels` in `StepProvider` (and `StepWrapper`, that also is strange). so we should not provide them in child components as props. Maybe better to delete `childrenArr`, prop `steps` from `StepWrapper ` and call child components directly in `StepWrapper` with access to the `StepProvider`. 
Discuss with team what we can do with this labels.

Reply:
I've refactored my code and removed `stepLabels` from `SubjectStep` component.
We'll discuss your suggestion with a team and maybe create a separate task for that improvement.
I don't want to do it in the scope of the current story to avoid potential conflicts. I think we should do it when the whole stepper is ready.